basic/error.dat

```diff
@@ -19,10 +19,13 @@ arraysimple3.c		0
 arraysimple4.c		0
 arraysimple5.c		0
 arraysimple6.c		0
+boundCheck5		0
```

Comment:
@rasoolmaghareh This should now be:
```
pointer_wp4.c 0
pointer_wp5.c 0
```
and preferably be moved right after `pointer_wp3.c ...`.